IDEAS/Buildings/Validation/Tests/ConvectionVerification.mo

```diff
@@ -6,16 +6,16 @@ model ConvectionVerification
   inner BoundaryConditions.SimInfoManager sim
     annotation (Placement(transformation(extent={{-92,68},{-82,78}})));
   IDEAS.Buildings.Validation.Cases.Case900 CaseLin(building(
-      roof(linearise_a=true),
-      wall(each linearise_a=true),
-      floor(linearise_a=true),
-      win(each linearise_a=true)))
+      roof(linIntCon_a=true),
```

Comment:
@Mathadon to add a unit test for this model

Reply:
So, this error I only spotted because I tried to check all models that used the Building template, to be sure I didn't create problems. This specific problem apparently comes from a change in the name of this parameter  years ago. So, I'm not even aware of the purpose of this one.

Reply:
This was just a reminder to myself to add a unit test for this model since it was not tested apparently, so the model has indeed been bugged in the library for ages :)